### DISCUSSION TITLE
Problems of satellite orbit lines color

### DISCUSSION THREAD

**Discussion by uilzzw** [Stellarium/stellarium#3594] [Q&A] [answered]
On the Linux system. After opening the satellite orbit line, the color setting the satellite orbit line does not take effect.
Only the gray lines, the color of this is not effective
![stellarium-20240116-164832974](https://github.com/Stellarium/stellarium/assets/17981936/4d7a9470-eab2-4b0c-b892-e64689c51f34)

System:Arch Linux
Graphics:Nvidia(545.29.06)
Stellarium Version:24.4

**Comment by alex-w** [comment, marked as answer]
The gray lines using for orbits of invisible satellites, e.g. in shadow of Earth. On the screenshot, I see definition of color for CSS, but I don't see CSS in the sky nor orbit line for CSS.

**Reply by uilzzw** [reply]
Thank you very much.After I asked the question, I searched for it. I noticed that too. Forgive my ignorance. I am a beginner
